test/api/services/AuditServiceSpec.scala

```diff
@@ -14,15 +14,15 @@
  * limitations under the License.
  */
 
-package v1.services
+package api.services
 
+import api.models.audit.AuditEvent
 import play.api.Configuration
 import play.api.libs.json.Json
 import uk.gov.hmrc.http.HeaderCarrier
 import uk.gov.hmrc.play.audit.http.connector.AuditResult.Success
 import uk.gov.hmrc.play.audit.http.connector.{AuditConnector, AuditResult}
 import uk.gov.hmrc.play.audit.model.ExtendedDataEvent
-import v1.models.audit.AuditEvent
```

Comment:
For some reason I can't place a comment in the required place. Ref Line 57: type annotation should be added for val event (AuditEvent[String]) here and for all the other tests in the file.